packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -3079,8 +3077,8 @@ export class ProjectService extends BaseService {
     async getWarehouseFields(
         user: SessionUser,
         projectUuid: string,
-        tableName: string,
-        schema?: string,
+        tableName?: string,
```

Comment:
reflects that the API endpoint takes these as query params now so could be missing